### PR TITLE
add census_build_cube.py example code

### PR DESCRIPTION
**Pull request by atolopko-czi**
If you think this example code is worth adding, I'll re-create as a proper notebook.

### CODE REVIEW

File: api/python/notebooks/census_build_cube.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                else:
+                    # Series.add() requires dense array, but maybe there's a better way to do this
+                    X_aggs['sum'] = X_aggs['sum'].sparse.to_dense()
+                    X_stats = X_stats.add(X_aggs, fill_value=0)
+
+                print(f"Batch agg rows={X_aggs.shape[0]}")
+                print(f"Total agg rows={X_stats.shape[0]}")
+
+            X_stats.to_pickle('census_cube.pkl')
+            print(X_stats)
```
Comment (atolopko-czi) [inline api/python/notebooks/census_build_cube.py:76]: If converted to a notebook, will provide some example queries of the cube.

**Review by bkmartinjr — APPROVED** [review]
LGTM.

I leave the question of notebook conversion to @pablo-gar

**Comment by atolopko-czi** [comment]
> Who would be the audience for the notebook?

Maybe no one. :) This code was the result of my own experimentation, so did not have a clear audience in mind. However, if we wanted to share how one might query the data akin to our Gene Expression™️ product, this provides the core aggregation logic. It would need rankit and other filtering logic to be added, of course. 

I'm hardly pushing to have this in the notebooks dir, so feel free to just reject this PR if not worth including, and I'll stash the code elsewhere.

> Also naive question here, is the cube materialized in memory and then is gone at the end of the execution?

 Currently, it's just pickled. But the idea is that we could provide a few Gene Expression™️ -like filters for demonstration purposes.

**Review by pablo-gar — APPROVED** [review]
Got it, yeah I'm not opposed to it being a notebook. Having what you just described as and intro to the notebook can be helpful. Eventually when go into utilizing the Cell Census for internal cases (e.g. Gene Expression) this can be a good reference but I think ultimately the engs will come to you and ask how to this or that.

I can see how it could also be helpful for other non cellxgene folks to start experimenting with aggregations.

**Comment by pablo-gar** [comment]
@atolopko-czi in light of recent conversations about keeping the API light and reducing maintenance for functions we won't support, I recommend we don't merge this and close it up.

any concerns with that?

**Comment by atolopko-czi** [comment]
Moved to a [gist](https://gist.github.com/atolopko-czi/c786485d0b8e851488b2c3aba219413a) for posterity

**Pull request closed without merging**